### DISCUSSION TITLE
Probe corner and Z too close

### DISCUSSION THREAD

**Discussion by rmaltais** [terjeio/ioSender#391] [Q&A] [answered]
Hi.
I would like to know if there is a way to move the probe a little less close to the edge of part. If I detect the corner of a part at the same time with height of the Z, it's often have less precise corners and I would like the probe to be less close to the edge. I can't find a way to move the probe other than manually after running the script in IoSender

**Comment by terjeio** [comment]
> I would like to know if there is a way to move the probe a little less close to the edge of part. 

Probe Z separately? IMO the only change that could be done is to enforce a minimum offset from the corner regardless of the probe diameter, but what should that be?

**Reply by rmaltais** [reply]
I would need at least 8mm in XY, which seems sufficient to me.

**Reply by rmaltais** [reply]
Obviously, I know that I can measure the Z independently but it is better with the XYZ probe to do it all at once. In case like my part which has a chamfer on the corners, my probe is too close. Maybe eventually be able to put the offset.

**Reply by terjeio** [reply, marked as answer]
I have added a field for _Edge offset_ in the latest [edge versions](https://www.io-engineering.com/downloads) (P10), can you try this as I have not done any extensive testing myself.
If the offset is set to zero 0.5x diameter of the probe tool is used (same as before), it might also be set negative. Note that I do not do any sanity check on the entered value, perhaps I should?

**Reply by rmaltais** [reply]
This offset is exactly what i want and work perfectly, but this new version can't IU jog fast and slow just with step. The JOG menu is always open. Maybe is a test version ?

**Reply by terjeio** [reply]
It is prerelease versions, you installed the XL variant?

**Comment by rmaltais** [comment]
No, what is feature of this version ?
________________________________
De : Terje Io ***@***.***>
Envoyé : dimanche 21 juillet 2024 00:49
À : terjeio/ioSender ***@***.***>
Cc : Diltech ***@***.***>; Author ***@***.***>
Objet : Re: [terjeio/ioSender] Probe corner and Z too close (Discussion #391)


It is prerelease versions, you installed the XL variant?

**Reply by terjeio** [reply]
* Allow use of modified legacy Grbl controller.
* Added support for spindle selection when multiple spindles are available.
* No longer stops jobtimer when changing tool.
* Added FTP port settings, available when controller supports FTP file transfer.
* Fixed and improved lathe turning wizard, taper mode.
* Improved handling of cycle start signal
* Improved tool length probing
* Added pre/post probing job macro support.
* Added Edge offset input in Edge finder tabs.
* A number of minor bug fixes and improvements.

Is the jog dialog overlaid the main window or part of it? It should be overlaid like this if not the XL variant:

![image](https://github.com/user-attachments/assets/ef3b440b-6896-409b-b247-dd3a7cf6ea55)

**Reply by rmaltais** [reply]
I installed XL version but jog windows always open and can't jog continously.

![ioSender_E36A4HrcRY](https://github.com/user-attachments/assets/9d4e3599-b0ea-4ac4-80c6-e05b5a3d6ce9)

**Reply by terjeio** [reply]
The XL version has the jog control as part of the main window, the non XL version has not (it is a "flyout" in that version).
Only keyboard jogging can be used for continuous jogging and that is disabled by default, a shorcut to the executable has to be added to enable it, [details here](https://github.com/terjeio/ioSender/wiki/Setup-and-configuration).
FYI the default behaviour was changed in december 2021 after user feedback.

**Comment by rmaltais** [comment]
oh! Yes sorry ! I forgot this option (long time used this on my other computer).Le 25 juill. 2024 à 00:04, Terje Io ***@***.***> a écrit :﻿
The XL version has the jog control as part of the main window, the non XL version has not (it is a "flyout" in that version).
Only keyboard jogging can be used for continuous jogging and that is disabled by default, a shorcut to the executable has to be added to enable it, details here.
FYI the default behaviour was changed in december 2021 after user feedback.
